jsk_topic_tools/CMakeLists.txt

```diff
@@ -168,27 +168,32 @@ install(TARGETS topic_buffer_server topic_buffer_client jsk_topic_tools
   ARCHIVE DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
   LIBRARY DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
   RUNTIME DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
-  )
+)
 
 install(FILES
   jsk_topic_tools_nodelet.xml
   jsk_topic_tools_test_nodelet.xml
-  DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION})
+  DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}
+)
 
 install(DIRECTORY include/${PROJECT_NAME}/
-  DESTINATION ${CATKIN_PACKAGE_INCLUDE_DESTINATION})
-# install(DIRECTORY include
-#   DESTINATION ${CATKIN_PACKAGE_INCLUDE_DESTINATION}
-#   )
+  DESTINATION ${CATKIN_PACKAGE_INCLUDE_DESTINATION}
+)
+
+install(DIRECTORY scripts/
+  DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
+  USE_SOURCE_PERMISSIONS
+)
 
 install(FILES ${jsk_topic_tools_test_files}
```

Comment:
`jsk_topic_tools_test_files` -> `jsk_tools_test_files` ?

Reply:
Yes, in jsk_tools/CMakeLists.txt, that variable was wrong and I fixed it.

But in jsk_topic_tools/CMakeLists.txt,

https://github.com/jsk-ros-pkg/jsk_common/blob/345bfb8c1f6eb658f3ac49121e1ec80a6cf73e4f/jsk_topic_tools/CMakeLists.txt#L125-L128

It seems that `jsk_topic_tools_test_files` is correct, isn't it?